spec/processors/jpeg2k_spec.rb

```diff
@@ -29,6 +29,9 @@
       @sample_cfg = YAML.load_file(File.expand_path('../../fixtures/jpeg2k_config.yml', __FILE__))['test']
       Hydra::Derivatives.kdu_compress_recipes = @sample_cfg['jp2_recipes']
     end
+    after(:all) do
```

Comment:
I could do :each but it might be overkill.  I was matching the before(:all) that sets kdu_compress_recipes which the reset undos.